polaris-core/src/main/java/org/apache/polaris/core/storage/azure/AzureStorageConfigurationInfo.java

```diff
@@ -19,20 +19,18 @@
 package org.apache.polaris.core.storage.azure;
 
 import com.fasterxml.jackson.annotation.JsonCreator;
-import com.fasterxml.jackson.annotation.JsonIgnore;
 import com.fasterxml.jackson.annotation.JsonProperty;
 import com.google.common.base.MoreObjects;
 import jakarta.annotation.Nonnull;
 import jakarta.annotation.Nullable;
 import java.util.List;
 import java.util.Objects;
+import org.apache.polaris.core.PolarisConfiguration;
+import org.apache.polaris.core.context.CallContext;
 import org.apache.polaris.core.storage.PolarisStorageConfigurationInfo;
 
 /** Azure storage configuration information. */
 public class AzureStorageConfigurationInfo extends PolarisStorageConfigurationInfo {
-  // technically there is no limitation since expectation for Azure locations are for the same
-  // storage account and same container
-  @JsonIgnore private static final int MAX_ALLOWED_LOCATIONS = 20;
 
```

Comment:
Since the 3 cloud providers had different values, how about different configuration params?

Reply:
See my comment here: https://github.com/apache/polaris/pull/1068#discussion_r1974117216